### ISSUE TITLE
Can you guide how to prepare the dataset for training your model?

### ISSUE THREAD

**Issue by IceIce1ce** [wuchangw/AnyAD#3]

**Comment by wuchangw** [comment]
You can refer to the following file structure：
├── train/
│   ├── flair/
│   │   ├── 001.png
│   │   ├── 002.png
│   │   └── ...
│   ├── t1/
│   │   ├── 001.png
│   │   ├── 002.png
│   │   └── ...
│   ├── t1ce/
│   │   └── ... 
│   └── t2/
│       └── ... 
└── test/
    ├── NORMAL/
    │   ├── flair/
    │   │   ├── 101.png
    │   │   └── ...
    │   └── t1/...
    └── ABNORMAL/
        ├── flair/
        │   ├── 201.png
        │   └── ...
        ├── t1/
        │   └── ... 
        └── mask/
            ├── 201_mask.png
            └── ...
The training set only needs normal, anomaly-free data, while the test set needs to include both normal and anomalous data, with 'Normal' and 'Abnormal' as the directory names. Note that anomalous data requires corresponding mask files. I believe you can deploy it quickly.

**Comment by IceIce1ce** [comment]
Hi author, thanks for your information.
